torch/testing/_internal/distributed/rpc/dist_autograd_test.py

```diff
@@ -845,6 +845,20 @@ def _verify_backwards_remote(self, tensors, context_id, local_grads, *args):
 
         self.assertEqual(ngrads, len(grads))
 
+    @dist_init
+    def test_backward_no_grad_on_tensor(self):
+        t1 = torch.rand((3, 3), requires_grad=True)
+        t2 = torch.rand((3, 3), requires_grad=True)
+        with dist_autograd.context() as context_id:
+            loss = rpc.rpc_sync(
+                "worker{}".format(self._next_rank()),
```

Comment:
nit: Id' suggest defining a method in `dist_utils` like `def worker_name(rank):` to avoid using the literal string.